stories/assets/js/filter_seach_bar.js

```diff
@@ -26,22 +26,22 @@ const toggleFilter = function () {
     }
     if (el.is(':checked')) {
       const optionValue = $(this).siblings().text();
-      const chips = $(currentChipsWrapper).find('.chips:first-child').clone();
+      const chips = $('<a class="chips chips__cross" href="#" role="button">demo</a>').clone();
```

Comment:
@Ashish-srij Can you change class names to just `chip` and `chip-close` respectively? Also remove `href` attribute if there is no link provided and kindly add `tabindex` for accessibility.
```
<a role="button" tabindex="0" class="chip chip-cross">demo</a>
```
Remove the `chips__no-cross` class as its not being used anywhere in CSS or JS. Just use the class `chip-cross` in case of close icon.

Reply:
@shishirphadke Made the changes as per our discussion.